llvm/lib/Target/AMDGPU/AMDGPUInstructionSelector.cpp

```diff
@@ -1478,10 +1478,17 @@ bool AMDGPUInstructionSelector::selectG_ICMP_or_FCMP(MachineInstr &I) const {
   if (Opcode == -1)
     return false;
 
-  MachineInstr *ICmp = BuildMI(*BB, &I, DL, TII.get(Opcode),
-            I.getOperand(0).getReg())
-            .add(I.getOperand(2))
-            .add(I.getOperand(3));
+  MachineInstrBuilder ICmp =
```

Comment:
I don't remember why we manually select these. Should make another attempt to move this to use the selection patterns

Reply:
The point of moving to the pattern is to delete all of this code, not have a redundant path